swarm/api/content_type_test.go

```diff
@@ -0,0 +1,43 @@
+package api
```

Comment:
same with this file - please remove and split into `server_test.go` and `api_test.go`